classes/template.php

```diff
@@ -301,13 +299,12 @@ public function generate_pdf(bool $preview = false, int $userid = null, bool $re
                 $filename = get_string('certificate', 'customcert');
             }
             
-            $sql = "SELECT course,fullname as full_name
-                    FROM {customcert} c, 
-	            {course} cr
-                    WHERE c.id = :templateid
-			  AND c.course = cr.id";
+            $sql = "SELECT course,fullname
```

Comment:
Please see https://docs.moodle.org/dev/SQL_coding_style#Indentation.

Reply:
I thought I have corrected it already, I missed this

Reply:
I think now it is right, "FROM {customcert} c, {course} cr"  is in one line and identation is correct. Hope now it is properly finished! Sorry for all the extra work